app/webpacker/controllers/dropdown_controller.js

```diff
@@ -4,25 +4,41 @@ export default class extends Controller {
   static targets = ["arrow", "menu"];
 
   connect() {
+    this.collapsedClass = this.arrowTarget.dataset.collapsedClass.split(" ");
+    this.expandedClass = this.arrowTarget.dataset.expandedClass.split(" ");
```

Comment:
Because we're using the uncommon spread operator here, let's make it clear that they are arrays:

```suggestion
    this.collapsedClasses = this.arrowTarget.dataset.collapsedClass.split(" ");
    this.expandedClasses = this.arrowTarget.dataset.expandedClass.split(" ");
```